index.html

```diff
@@ -0,0 +1,67 @@
+
+<!DOCTYPE html>
+
+<html lang="en">
+    <link rel="stylesheet" href="styles.css"> 
+  <header>
+  <h1 id="title">TWO RIVERS MALL</h1>
+  </header>
+  <body>
+  <p id="description"> Fill the survey below about your experience at The two rivers mall</p>
+  <form id="survey-form">
+  <div class="input-headings">
+  
+    <label for="name" id="name-label">Name </label>
```

Comment:
following the project requirement using semantic `ul` &` li` tags. they are all list items and are better displayed this way. 👍

Reply:
- [ ] This is a great comment, `ul` and `li` are semantic best practices, however, it will be more efficient if you clearly explain to the student the `why` you are asking for the change, I highly recommend checking this [Why should I use UL/LI in my HTML?](https://stackoverflow.com/questions/13416866/why-should-i-use-ul-li-in-my-html) it will give you and idea on how you should answer the `why` for this requested changes 👍🏼